Microsoft.Toolkit.Uwp.SampleApp/SamplePages/InAppNotification/InAppNotificationPage.xaml.cs

```diff
@@ -57,6 +58,15 @@ private void Load()
                 _exampleInAppNotification?.Show(GetRandomText(), NotificationDuration);
             });
 
+            SampleController.Current.RegisterNewCommand("Show notification with object", (sender, args) =>
```

Comment:
We should also update the code file that goes with the sample to show the way to call the Show method like this too.

Reply:
Done.
I've also updated the documentation in [Doc PR #365](https://github.com/MicrosoftDocs/WindowsCommunityToolkitDocs/pull/365)